src/full/Agda/TypeChecking/Unquote.hs

```diff
@@ -756,8 +756,7 @@ evalTCM v = Bench.billTo [Bench.Typing, Bench.Reflection] do
     tcCommit :: UnquoteM Term
     tcCommit = do
       dirty <- gets fst
-      when (dirty == Dirty) $
-        liftTCM $ typeError $ GenericError "Cannot use commitTC after declaring new definitions."
+      when (dirty == Dirty) $ liftTCM $ unquoteError CommitAfterDef
```

Comment:
The `liftTCM` is likely superfluous since `unquoteError` is polymorphic in the monad, isn't it?